src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.ts

```diff
@@ -538,7 +538,7 @@ export class BadgeClassEditFormComponent extends BaseAuthenticatedRoutableCompon
 				});
 			})
 			.catch((error) => {
-				this.messageService.reportAndThrowError('Failed to obtain ai skills.', error);
+				this.messageService.reportAndThrowError(`Failed to obtain ai skills: ${error.message}`, error);
```

Comment:
What about updating error msg to `Please write a longer content description to get competencies suggestions`? could be in German as well.

Reply:
The error message is contained in `error.message` since it is passed in the backend (*if* you use the correct backend version!)